### ISSUE TITLE
`*_loader_dependencies.json` doesn't work

### ISSUE THREAD

**Issue by DragonEggBedrockBreaking** [QuiltMC/quilt-loader#66]
I have tested both `fabric_loader_dependencies.json` and `quilt_loader_dependencies.json`, and neither of them work. I have tested them with the basic configuration of:

```json
{
    "version": 1,
    "overrides": {
        "tiefix": {
            "-depends": {
                "fabric-resource-loader-v0": "IGNORED"
            }
        }
}
```

but I still get an error on launch declaring a missing fabric resource loader dependency. This occurs with any other mods/dependencies I have tried it on.

The files were placed in `**/.minecraft/config/`, as with fabric.

On fabric, as expected, `fabric_loader_dependencies.json` works fine.

@wafflecoffee mentioned on the QuiltMC discord server that she had some issues too.

**Comment by AlexIIL** [comment]
Yes, dependency overrides haven't been added for quilt mods yet (and were disabled for fabric mods), so this is expected (but still a bug).

**Comment by AlexIIL** [comment]
Sorry, I should have commented much ealier. I plan to write an RFC for improving on the `fabric_loader_dependencies.json` format for quilt's overrides, but I don't plan to change fabric's format. As such only the `quilt_loader_dependencies.json` part should be assigned to me, and anyone else could implement fabric's.

**Comment by Julienraptor01** [comment]
This feature would be really useful to have currently for all the 1.18 mods that work in 1.19 but have 1.18 specified in their *.mod.json because patching all of them manually is a pain. And it's a big problem when sharing a modpack when you don't want to just give jars.

**Comment by DioEgizio** [comment]
Wanted to say that I think this is a very nice fabric feature and I'd like to have it on Quilt too

**Comment by comp500** [comment]
Note that loader 0.18.1 implemented a similar quilt-specific overrides file, see [Dependency Overrides](https://github.com/QuiltMC/quilt-loader/wiki/Dependency-Overrides)

**Comment by DragonEggBedrockBreaking** [comment]
Closing based on comp500's comment
